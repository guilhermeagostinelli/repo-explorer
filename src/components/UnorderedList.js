import styled from "styled-components";

const UnorderedList = styled.ul`
  list-style: none;
  padding: 0;
`;

export default UnorderedList;
