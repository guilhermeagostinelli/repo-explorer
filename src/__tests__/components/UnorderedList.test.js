import React from "react";
import { create } from "react-test-renderer";
import UnorderedList from "../../components/UnorderedList";
import "jest-styled-components";

describe("<UnorderedList />", () => {
  it("should match the snapshot", () => {
    const unorderedList = create(<UnorderedList />);
    expect(unorderedList.toJSON()).toMatchSnapshot();
  });
});
