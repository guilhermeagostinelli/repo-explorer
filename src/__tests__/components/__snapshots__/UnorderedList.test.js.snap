// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UnorderedList /> should match the snapshot 1`] = `
.c0 {
  list-style: none;
  padding: 0;
}

<ul
  className="c0"
/>
`;
